oe_contact_forms.module

```diff
@@ -203,6 +214,62 @@ function oe_contact_forms_form_contact_form_form_alter(&$form, FormStateInterfac
   $form['#entity_builders'][] = '_oe_contact_forms_contact_form_builder';
 }
 
+/**
+ * Gets the URI without the 'internal:' or 'entity:' scheme.
+ *
+ * The following two forms of URIs are transformed:
+ * - 'entity:' URIs: to entity autocomplete ("label (entity id)") strings;
+ * - 'internal:' URIs: the scheme is stripped.
+ *
+ * It's sad that LinkWidget::getUriAsDisplayableString() is a protecgted method,
+ * and we can't benefit from its implementation.
+ *
+ * @param string $uri
+ *   The URI to get the displayable string for.
+ *
+ * @return string
+ *
+ * @see \Drupal\link\Plugin\Field\FieldWidget\LinkWidget::getUriAsDisplayableString()
+ */
+function _oe_contact_forms_get_privacy_default($uri): string {
+  if (!$uri) {
+    return '';
```

Comment:
How can this happen? Also, no type hinting of the `$uri` param?

Reply:
your right, its a required field, cant, ill remove it